core/domain/exp_services_test.py

```diff
@@ -447,16 +447,10 @@ def mock_delete_docs(doc_ids, index):
         with delete_docs_swap:
             exp_services.delete_exploration(self.owner_id, self.EXP_ID)
 
-    def test_create_new_exploration_error_cases(self):
+    def test_no_errors_are_raised_when_creating_default_exploration(self):
```

Comment:
Added two tests: one to editor_test.py to check that default explorations can't be published, and another to exp_services_test.py to check that they fail strict validation.